_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why is it so important to do pre-GSoC analysis and
+ design as early as possible, to be finalized in the cEP?"
```

Comment:
If `to be finalized` can be replaced by `which is finalized` without changing the meaning, then it would probably be a better choice.

Reply:
using `which is possible` is not conveying the meaning what actually we want to address and also, using this is making some sort of discontinuity between `... design as early as possible` and `which is finalised ...`.
So, `to be finalized` seems to be a good choice.